digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "NotesMaster",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://notesmaster.com/termsofuse"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "System requirements - https://notesmaster.com/systemrequirements"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Federal Children’s Online Privacy Protection Act of 1998",
+      "U.S. Family Educational Rights and Privacy Act"
+    ],
+    "adherenceSteps": [
+      "https://notesmaster.com/privacy"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "W3C Accessibility Standards",
+      "WCAG 2.1 standards",
+      "Creative Commons standards"
+    ],
+    "evidenceStandardSupport": [
+      "Please see reference to our Terms of Use which highlights our adherence to the Creative Commons Standards, as well as our adherence to the the Federal Children’s Online Privacy Protection Act of 1998 (“COPPA”) in the United States and the European Union’s General Data Protection Regulation (GDPR) of 2018: https://notesmaster.com/termsofuse"
```

Comment:
See comment above for `standardsList`. Please rephrase to remove mention to Creative Commons Standards or clarify.

Reply:
Removed in commit 7eda5ff